linux_os/guide/system/network/network-ufw/set_ufw_default_rule/rule.yml

```diff
@@ -0,0 +1,39 @@
+documentation_complete: true
+
+prodtype: ubuntu2004,ubuntu2204
+
+title: 'Ensure ufw default deny firewall policy'
```

Comment:
In alignment to [Style Guide](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#rule-sections) it must be in [Title case](https://en.wikipedia.org/wiki/Title_case)